src/Sylius/Bundle/CoreBundle/Fixture/OrderFixture.php

```diff
@@ -193,6 +197,7 @@ private function generateItems(OrderInterface $order): void
                 /** @var OrderItemInterface $item */
                 $item = $generatedItems[$variant->getCode()];
                 $this->orderItemQuantityModifier->modify($item, $item->getQuantity() + random_int(1, 5));
+                $variant->setShippingRequired($shippingMethodsAvailable);
```

Comment:
I think it should be an option for product fixtures and we shouldn't change previous items in other fixtures.

Reply:
Yeah, we shouldn't change variants in order fixture.